src/api/app/controllers/webui/cloud/configurations_controller.rb

```diff
@@ -4,6 +4,7 @@ class ConfigurationsController < WebuiController
       before_action :set_breadcrumb
 
       def index
+        switch_to_webui2
```

Comment:
Please, add the condition until we finish the migration:

```
switch_to_webui2 if Rails.env.development?
```

Reply:
done